src/Chainweb/RestAPI/Utils.hs

```diff
@@ -518,7 +518,7 @@ bindPortTcp p interface = do
         socket <- bindPortGen N.Stream (int p) interface
         port <- N.socketPort socket
         return (int port, socket)
-    N.listen sock (max 2048 N.maxListenQueue)
+    N.listen sock (min 2048 N.maxListenQueue)
```

Comment:
It looks like `N.maxListenQueue` is just fine. If it's not sufficient we would have to adjust `/proc/sys/net/core/somaxconn`.

```suggestion
    N.listen sock N.maxListenQueue
```